ChangeLog.d/9126.txt

```diff
@@ -0,0 +1,6 @@
+Default behavior changes
+   * In a pure PSA client build (i.e. MBEDTLS_PSA_CRYPTO_CLIENT &&
```

Comment:
I'm sorry, I just don't understand this sentence.

Taking the perspective of a user: No support will be provided for something? So we're fixing the lack of support for something? But what is it? What do client-only builds have to do with drivers — don't drivers live on the server side?

Knowing what this is about: I still don't see how drivers are involved. What we're fixing is auto-enabling some built-in algorithms. So many people (amongst people who use client-only builds) benefit from a code size improvement (a non-functional improvement which alone would normally not justify a changelog entry), but some applications may break if they were accidentally relying on the auto-enablement, which is why this does need a changelog entry.